lib/tasks/demo_data.rake

```diff
@@ -219,6 +219,23 @@ namespace :demo_data do
 
     brokerage.sync
 
+    crypto = Account.find_or_create_by(name: "Bitcoin Account") do |a|
+      a.family = family
+      a.accountable = Account::Crypto.new
+      a.balance = 10000
+      end
+
+    crypto_valuations = [
+    { date: 1.year.ago.to_date, value: 9000 },
+    { date: 200.days.ago.to_date, value: 9500 },
+    { date: 100.days.ago.to_date, value: 9444.96 },
+    { date: 20.days.ago.to_date, value: 10000 }
+    ]
```

Comment:
For a sample account like Bitcoin, we'll want to make sure and specify the currency, `BTC` and adjust these values to be smaller so that when it is converted to the family's currency (likely USD), it will represent a reasonable proportion of the portfolio.  I'd say somewhere around `0.1` is probably appropriate for a demo account (~$6,500 worth of BTC)